rest-api/src/main/resources/paths/studies/v3_studies_self_emails_resendVerify.yml

```diff
@@ -0,0 +1,27 @@
+post:
+    operationId: resendVerifyEmail
+    summary: Resends the verification email for the study's email.
+    description: |
+        Resends the verification email for the current study's email.
+    tags: 
+        - Studies
+        - _For Developers
+    security:
+        -   BridgeSecurity: []
+    parameters:
+        - name: type
+          description: Type of email that should resend email verification.
+          required: true
+          in: query
+          type: string
+          # Our version of Swagger doesn't support enums in query parameters, so we have to list out enum values
+          # manually. Unfortunately, there is no compile-time validation of this, and the JavaSDK is generated as a
+          # String.
+          enum: [CONSENT_NOTIFICATION]
```

Comment:
I don't see where CONSENT_NOTIFICATION is defined... ?

Reply:
Changed.